test/integration/studios-page.test.js

```diff
@@ -52,5 +68,83 @@ describe('studio page while signed out', () => {
         let descriptionText = await studioDescription.getText();
         await expect(descriptionText).toEqual('a description');
     });
+});
+
+describe('studio management', () => {
+    // These tests all start on the curators tab of a studio and signed out
+
+    beforeAll(async () => {
+        driver = await buildDriver('www-integration studio management');
+        await driver.get(rootUrl);
+
+        // create a studio for tests
+        await signIn(username2, password, driver);
+        await findByXpath('//span[contains(@class, "profile-name")]');
+        await driver.get(rateLimitCheck);
+        await driver.get(myStuffURL);
+        await clickXpath('//form[@id="new_studio"]/button[@type="submit"]');
+        await findByXpath('//div[@class="studio-tabs"]');
+        promoteStudioURL = await driver.getCurrentUrl();
+        curatorTab = await promoteStudioURL + 'curators';
+    });
 
+    beforeEach(async () => {
+        await clickXpath('//a[contains(@class, "user-info")]');
+        await clickText('Sign out');
+        await driver.get(curatorTab);
+        await findByXpath('//div[@class="studio-tabs"]');
+    });
+
+    afterAll(async () => await driver.quit());
+
+    test('invite a curator', async () => {
+        // sign in as user2
+        await signIn(username2, password, driver);
+        await findByXpath('//span[contains(@class, "profile-name")]');
+
+        // invite user3 to curate
+        let inviteBox = await findByXpath('//div[@class="studio-adder-row"]/input');
+        await inviteBox.sendKeys(username3);
+        await clickXpath('//div[@class="studio-adder-row"]/button');
+        let inviteSuccess = await findByXpath('//div[@class="alert-msg"]'); // the confirm alert
```

Comment:
this could be seeing an error alert instead (which also has the alert-msg class), so we might consider changing this to be more specific to the success alert